session.go

```diff
@@ -283,6 +283,18 @@ func (s *Session) init() error {
 	return nil
 }
 
+// AwaitSchemaAgreement will query the state of the schema versions across
+// all nodes of the cluster. The maximum amount of time this takes is governed
+// by the MaxWaitSchemaAgreement setting in the configuration (default: 60s).
+func (s *Session) AwaitSchemaAgreement(ctx context.Context) error {
+	if s.cfg.disableControlConn {
+		return errNoControl
+	}
+	return s.control.withConn(func(conn *Conn) *Iter {
```

Comment:
I didn't use `s.control.awaitSchemaAgreement()` here because I wanted the context to be honored and used. I hope that's okay. I can change it if you want.

Reply:
Looks okay to me.

Reply:
Very good, thank you!